### DISCUSSION TITLE
speed up open web without loading splash screen

### DISCUSSION THREAD

**Discussion by bobwatcherx** [flet-dev/flet#2548] [Q&A]
### Question

when run in the browser, flet opens a logo splash screen. Is there a way to speed it up like opening a regular html file? when after building

### Code sample

```python
no code
```

### Error message

```bash
no
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by Daxexs** [comment]
> ### Pregunta
> cuando se ejecuta en el navegador, flet abre una pantalla de presentación con el logotipo. ¿Hay alguna manera de acelerarlo como abrir un archivo html normal? cuando después de construir
> 
> ### Ejemplo de código
> ```python
> no code
> ```
> 
> ### Mensaje de error
> ```shell
> no
> ```
> 
> ### -------------------------------------------------- ----
> * [x] He buscado respuestas a mi pregunta tanto en los números como en discusiones anteriores.

If there is, to do this you must customize the index.html file located in the assets folder, there will still be loading at the beginning of loading the website, since it is by default. This will allow you to customize the logo or remove it, more information -> https://flet.dev/docs/guides/python/deploying-web-app/customizing-web-app

**Comment by FeodorFitsner** [comment]
Use `--no-web-splash` option when building with `flet build`: https://flet.dev/docs/guides/python/packaging-app-for-distribution#splash-screen

**Comment by vindevoy** [comment]
What I did, and it saves a couple of seconds, is not to do this:

import flet as ft

but do imports from flet_core where you only import the control you need

from flet_core import AppView, ......
